### PR TITLE
Rails 4/5 compatibility

### PR DESCRIPTION
**Pull request by cmckni3**
* Use stabby lambda for scopes
* Remove `attr_accessible`
* Ruby 2 hash syntax

### CODE REVIEW

File: lib/generators/my_zipcode_gem/templates/zipcode_model.rb [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+
   scope :without_county, -> { where("county_id IS NULL") }
   scope :without_state, -> { where("state_id IS NULL") }
   scope :ungeocoded, -> { where("lat IS NULL OR lon IS NULL") }
 
   class << self
     def find_by_city_state(city, state)
-      includes(:county => :state).where("city like '#{city}%' AND states.abbr like '%#{state}%'").first
+      includes(county: :state)
+        .where("city like ? AND states.abbr like ?", "#{city}%", "%#{state}%")
```
Comment (cmckni3) [inline lib/generators/my_zipcode_gem/templates/zipcode_model.rb]: I don't care for this rule

File: lib/generators/my_zipcode_gem/templates/county_model.rb [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-  validates :name, :uniqueness => {:scope => :state_id, :case_sensitive => false}, :presence => true
-  
-  scope :without_zipcodes, joins("LEFT JOIN zipcodes ON zipcodes.county_id = counties.id").where("zipcodes.county_id IS NULL")
-  scope :without_state, where("state_id IS NULL")
-  
+
+  validates :name, uniqueness: { scope: :state_id, case_sensitive: false },
+                   presence: true
+
+  scope :without_zipcodes, -> { joins("LEFT JOIN zipcodes ON zipcodes.county_id = counties.id").where("zipcodes.county_id IS NULL") }
```
Comment (midwire) [inline lib/generators/my_zipcode_gem/templates/county_model.rb]: Someday I'd like to work out the Arel for this, but I have too much to do for today.

Comment (cmckni3) [inline lib/generators/my_zipcode_gem/templates/county_model.rb]: I can change it in a separate pull request. I'm not sure why I can't run the tests now. Might have a newer version of one of the testing libraries.

Comment (midwire) [inline lib/generators/my_zipcode_gem/templates/county_model.rb]: No worries. For some reason running tests locally fails for me as well.

**Pull request merged**